### PR TITLE
Create Edit Event

### PR DESCRIPTION
**Pull request by farellano0**
## Quick Info
Create edit event interface and their styles

## What does this change?
Modify styles of buttons.
Add the edit page to edit events.

## Why are you changing that?
Because some problems appeared with the buttons and wanted to apply the best coding practices.
A user interface was needed to modify an event.

## Screenshots
image

image

image

image

### CODE REVIEW

File: app/assets/stylesheets/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.container {
+  margin: 1rem;
+  width: fit-content;
+}
```
Comment (juperameza) [inline app/assets/stylesheets/base/_base.scss]: Add new line at the end of this file please

File: app/assets/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -6,41 +6,82 @@
...
+// }
+
+// .btn-return {
+//   align-items: center;
+//   background-color: $gray;
+//   border-radius: 10px;
+//   border: 1px solid $blue;
+//   color: $black;
+//   display: flex;
+//   height: fit-content;
```
Comment (juperameza) [inline app/assets/stylesheets/components/_buttons.scss]: Dont leave commented code in github

File: app/views/bright_events/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <%= form_for (@event) do |f| %>
+      <%= f.text_field :title, placeholder: 'Título', value: @event.title, required: true %>
+      <%= f.text_field :description, placeholder: 'Descripción', value: @event.description, required: true %>
+      <%= f.date_field :date, placeholder: 'Fecha', value: @event.date, min: 0.days.ago, required: true %>
+      <%= f.text_field :location, placeholder: 'Ubicación', value: @event.location, required: true %>
+      <%= f.number_field :cost, in: 0.00..1000000.00, step: 0.01, placeholder: 'Precio', value: @event.cost, required: true %>
+      <%= f.submit 'Guardar cambios', class: 'btn btn-submit btn-warning', required: true %>
+    <% end %>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/bright_events/edit.html.erb]: Have you considered doing a partial for the form since is the same code here and in the new ???
Maybe do it in another pr

**Review by juperameza — CHANGES_REQUESTED** [review]

**Comment by MauricioNC** [comment]
I agree with the requested changes of Juan, please keep an eye and apply the new changes.

File: app/assets/stylesheets/base/_base.scss (hunk 1 above)
Comment (Zama15) [reply app/assets/stylesheets/base/_base.scss]: done

File: app/assets/stylesheets/components/_buttons.scss (hunk 2 above)
Comment (Zama15) [reply app/assets/stylesheets/components/_buttons.scss]: okay

File: app/views/bright_events/edit.html.erb (hunk 3 above)
Comment (Zama15) [reply app/views/bright_events/edit.html.erb]: little tricky, but already implemented :+1:

**Comment by Zama15** [comment]
> I agree with the requested changes of Juan, please keep an eye and apply the new changes.

already done everything :+1:

**Review by juperameza — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]
